src/sqlfluff/core/templaters/python.py

```diff
@@ -175,7 +175,7 @@ def infer_type(s):
         except (SyntaxError, ValueError):
             return s
 
-    def get_context(self, fname=None, config=None) -> Dict:
```

Comment:
Uninteresting changes updating the Python templater to align with the changes to `jinja.py`.